students/eowyn/session10/ff-mailroom/test_ffmailroom.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+
+import ffmailroom as mr
+import os.path
+
+
+def test_donor_init():
+    name = "Ada Lovelace"
+    amt = 345
+    d = mr.Donor(name)
+    d.add_donation(amt)
+    assert d.name == name
+    assert d.donations == [345]
+
+def test_name():
+    name = "Ada Lovelace"
+    amt = 345
+    d = mr.Donor(name)
+    d.add_donation(amt)
+    assert d.name == name
```

Comment:
you already tested this in test_donor_init --no need to do it again. (not that it hurts, either)